pkg/projects/dir.props

```diff
@@ -96,6 +96,38 @@
     </BuildRID>
   </ItemGroup>
 
+  <!-- ================== Section for Universal Windows Platform package ================== -->
```

Comment:
This is fixed now, as we agreed offline, I separated the package rids into individual props files that can be overwritten.